PlayCover/fr.lproj/Localizable.strings

```diff
@@ -1,96 +1,115 @@
+"app.update" = "Mettre à jour l'app";
+
+"app.install.unzip" = "Extraction de l'app...";
+"app.install.createWrapper" = "Création du conteneur de l'application...";
+"app.install.installPlayTools" = "Installation de PlayTools...";
+"app.install.signing" = "Signature de l'app...";
+"app.install.addToLib" = "Ajout de l'app dans la librairie...";
+"app.install.copy" = "Copie de l'app...";
+"app.install.finished" = "Fini";
+
+"search.search" = "Recherche en cours...";
+
+"alert.appCacheCleared" = "Les caches de l'app ont été effacés!";
+"alert.kmImported" = "Keymap importé!";
+"alert.errorImportKm" = "Erreur lors de l'import";
+"alert.app.delete" = "Toutes données de l'app vont être effacées. Vous aurez peut être besoin de retélécharger des fichiers de l'app. Voulez-vous continuer?";
+"alert.wrongFileType" = "Mauvais type de fichier";
+"alert.wrongFileType.message" = "Choisir un fichier ipa"; 
+"alert.followInstructionAndRestartApp" = "Veuillez suivre les instructions données, et redémarrer l'app.";
+"alert.error" = "Un erreur a occuré"; 
+"alert.success" = "Succès!"; 
+"alert.moveAppToApplications" = "PlayCover doit être dans le dossier Applications. Appuyer le bouton ci-dessous pour automatiquement déplacer PlayCover à /Applications.";
+"alert.moveAppToApplications.move" = "Déplacer vers /Applications";
+
+"button.OK" = "OK";
+"button.Proceed" = "Continuer"; 
+"button.Cancel" = "Annuler";
+"button.Close" = "Fermer"; 
+"button.Dismiss" = "Fermer";
+"button.Install" = "Installer";
+
+"error.corruptedIPA" = "Ce fichier .ipa est corrompu. Il ne contient pas de fichier 'Info.plist'.";
+"error.waitInstallation" = "Veuiller attendre que l'installation actuelle termine."; 
+"error.reinstallApp" = "Veuillez réinstaller PlayCover";
+
+"menubar.downloadMoreApps" = "Télécharger plus d'apps"; 
+"menubar.downloadMoreApps.help" = "Utilisez ce site pour télécharger des apps"; 
+"menubar.exportToSideloady" = "Exporter pour Sideloadly";
+"menubar.exportToSideloady.help" = "Exporter des IPA pour Sideloady du site sideloady.io qui n'a pas besoin d'avoir SIP désactivé pour que certains apps marchent."; 
+
+"bottomBar.notices" = "Notes";
+"bottomBar.notices.help" = "Nouvelles et annonces des développers";
+"bottomBar.setupViewButton" = "Est-ce que vous rencontrez des problèmes?";
+
+"failedToFetchNotices" = "Échec à trouver les avis. Veuiller vérifier votre connection."; 
+
+"logs.copied" = "Registre copié"; 
+
+"enable.km" = "Activer le mappage de touches"; 
+"enable.gaming" = "Mode de mappage de clavier amélioré";
+"enable.km.info" = "Permet d'utiliser la souris et le keymap en même temps."; 
+"enable.adaptiveDisplay" = "Affichage adaptatif"; 
+"enable.adaptiveDisplay.info" ="Adapte la fenêtre de l'app pour qu'elle rentre dans votre moniteur.";
+"enable.jbBypass" = "Activer l'évitement Jailbreak (Alpha)"; 
+"enable.jbBypass.info" = "Essaye d'éviter les détections de Jailbreak dans certains jeux (No guarentee to work all the time)";
```

Comment:
No guarentee to work all the time -> Aucune garantie de fonctionnement

Reply:
Oh, I totally missed that one